docs/subprojects.rst

```diff
@@ -1,18 +1,42 @@
 Subprojects
 ===========
 
-We support the concept of Subprojects.
-If you add a subproject to a project,
-that documentation will also be served under the parent project's subdomain.
+Projects can be configured in a nested manner, by configuring a project as a
+*subproject* of another project. This allows for documentation projects to share
+a search index and a namespace or custom domain, but still be maintained
+independently.
 
-For example,
-Kombu is a subproject of celery,
-so you can access it on the `celery.readthedocs.io` domain:
+For example, a parent project, ``Foo`` is set up with a subproject, ``Bar``. The
+documentation for ``Foo`` will be available at:
 
-http://celery.readthedocs.io/projects/kombu/en/latest/
+https://foo.readthedocs.io/en/latest/
 
-This also works the same for CNAMEs:
+The documentation for ``Bar`` will be available under this same path:
 
-http://docs.celeryproject.org/projects/kombu/en/latest/
+https://foo.readthedocs.io/projects/bar/en/latest/
```

Comment:
I like the idea of using generic projects (foo and bar) in the general idea of the explanation. Maybe we can have another section with real life examples that point to projects and sub-projects so the user can see how they really work. If we don't control them, we can add a note like: "it may fails -said Tusam" :)